src/openforms/authentication/contrib/yivi_oidc/tests/test_auth_flow_init.py

```diff
@@ -0,0 +1,234 @@
+import base64
+import json
+import re
+
+from django.urls.base import reverse_lazy
+
+from furl import furl
+
+from openforms.authentication.constants import AuthAttribute
+from openforms.authentication.contrib.yivi_oidc.tests.base import mock_yivi_config
+from openforms.authentication.tests.factories import AttributeGroupFactory
+from openforms.authentication.tests.utils import URLsHelper
+from openforms.forms.tests.factories import FormFactory
+
+from ..models import YiviOpenIDConnectConfig
+from .base import IntegrationTestsBase
+
+
+class YiviInitTests(IntegrationTestsBase):
+    """
+    Test the outbound part of OIDC-based Yivi authentication.
+    """
+
+    CALLBACK_URL = f"http://testserver{reverse_lazy('oidc_authentication_callback')}"
+
+    def _assert_condiscon_scope(self, scope: str, expected: list[list[list[str]]]):
```

Comment:
```suggestion
    def assertCondisconScope(self, scope: str, expected: list[list[list[str]]]):
```
within this test case, it's just public API - you're free to define your own assertion helpers (and we use the same existing unittest.TestCase assert naming style)